packages/core/admin/admin/src/content-manager/components/ListViewTable/BulkActionButtons/index.js

```diff
@@ -176,6 +208,126 @@ const ConfirmDialogDeleteAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading
 
 ConfirmDialogDeleteAll.propTypes = confirmDialogsPropTypes;
 
+/* -------------------------------------------------------------------------------------------------
+ * BoldChunk
+ * -----------------------------------------------------------------------------------------------*/
+
+const BoldChunk = (chunks) => <Typography fontWeight="bold">{chunks}</Typography>;
+
+/* -------------------------------------------------------------------------------------------------
+ * SelectedEntriesTableContent
+ * -----------------------------------------------------------------------------------------------*/
+
+const SelectedEntriesTableContent = () => {
+  const { formatMessage } = useIntl();
+  const { selectedEntries, setSelectedEntries, rows } = useTableContext();
+
+  // Select all entries by default
+  React.useEffect(() => {
+    setSelectedEntries(rows.map((row) => row.id));
+  }, [setSelectedEntries, rows]);
+
+  return (
+    <>
+      <Typography>
+        {formatMessage(
+          {
+            id: getTrad('containers.ListPage.selectedEntriesModal.selectedCount'),
+            defaultMessage: 'test',
+          },
+          {
+            count: selectedEntries.length,
+            b: BoldChunk,
+          }
+        )}
+      </Typography>
+      <Box marginTop={5}>
+        <Table.Content>
+          <Table.Head>
+            <Table.HeaderCheckboxCell />
+            <Table.HeaderCell fieldSchemaType="number" label="id" name="id" />
+            <Table.HeaderCell fieldSchemaType="string" label="name" name="name" />
+          </Table.Head>
+          <Tbody>
+            {rows.map((entry, index) => (
+              <Tr key={entry.id}>
+                <Body.CheckboxDataCell rowId={entry.id} index={index} />
+                <Td>
+                  <Typography>{entry.id}</Typography>
+                </Td>
+                <Td>
+                  <Typography>{entry.name}</Typography>
```

Comment:
I would say use the Entity title instead the same way we do for relations and if it does not exist fall back to showing the ID only

Reply:
Good catch, I think we should check with product about removing any other attributes and just have id + validation status. I feel like otherwise it needs to be configurable like the other table where users can decide what attributes they want to see.

Reply:
Works for me. What's the entity title and how do you configure it? 🐘

I didn't find anything in the docs.

Reply:
edit view
configure view 
At the top directly bellow Settings you see Entry title with a link to the field that it will display.

Reply:
Nice thanks!

Reply:
I updated the PR with this change. Sometimes the main field can be the ID, so I hid it in this case.